h2o-core/src/main/java/hex/Model.java

```diff
@@ -1072,6 +1072,11 @@ static protected void cleanup_adapt( Frame adaptFr, Frame fr ) {
     return names;
   }
 
+  /** Allow subclasses to define their own BigScore class. */
+  protected BigScore makeBigScore(String[][] domains, String[] names , Frame adaptFrm, boolean computeMetrics, Job j) {
+    return new BigScore(domains[0],names.length,adaptFrm.means(),_output.hasWeights() && adaptFrm.find(_output.weightsName()) >= 0,computeMetrics, true /*make preds*/, j).doAll(names.length, Vec.T_NUM, adaptFrm);
```

Comment:
THIS IS the big PROBLEM! 🔥 

the task is created, launched, but code below it relaunched it again!